apps/nova/src/app/[locale]/(dashboard)/[wsId]/challenges/challengeCard.tsx

```
@@ -12,7 +12,13 @@ import {
 } from '@tuturuuu/ui/card';
 import { ArrowRight, Star } from 'lucide-react';
 import React, { useEffect, useState } from 'react';
-
+interface Challenge {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Challenge'. Consider to delete it or rename it.

'Challenge' is defined here:


(lint/suspicious/noRedeclare)